### PR TITLE
Screamtastic

### PR DESCRIPTION
**Pull request by Fox-McCloud**
Credit to to Goon for the sounds and the idea behind pitched-adjusted sounds by age.

I was shocked at how many people requested this of me.

- Changes sounds to allow you to specify a pitch for a sound
 - For coders: you must have vary set to 1 and "pitch" var set for this to work.
 - Whoever handled random pitch generation did it TWICE...whyyyy
- Refactors screams to be based on the species datums instead of based on snowflake checks
- implements Goon's more realistic sounding scream sounds for most species (Vox are still the same)
- Implement's Goon's monkey screaming sound for all monkey and subtypes
- Scream sound vary for male and female (non-monkey/Vox only); they're also pitch adjusted based on age
- Refactors emote cooldown handling to allow for longer cooldowns based on a specific emotes
- Screaming emote cooldown upped from 2 seconds to 5 seconds
- Borgs can now scream (this utilizes Goon's borg screaming sound)
- Removes old scream sound
- Removes chance for random Whilhelm scream
- IPCs scream like cyborgs

:cl: Fox McCloud
rscadd: Screaming has different sounds based on being male or female
soundadd: Implements Goon's screaming sounds.
rccadd: Screaming tone is now based on age of character instead of being random
tweak: Ups the cooldown on screaming from 2 seconds to 5 seconds
tweak: Removes chance for Whilhelm scream
rcsadd: Borgs can now scream
rcadd: Monkey's now have a unique screaming sound
rscadd: IPCs now scream like cyborgs
/:cl:

### CODE REVIEW

**Comment by TheDZD** [comment]
@tigercat2000 RIP your ears.

**Comment by NTSAM** [comment]
MYAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAH!!!!!!!!

:+1: my fucking headphones are gonna break blissfully

**Comment by taukausanake** [comment]
I can only scream every five seconds now? I know spam is one thing but getting someone's attention via screaming, which I did do once before, is pretty helpful. I think two seconds would be nice for that

**Comment by Fox-McCloud** [comment]
@taukausanake 

The sound has a longer duration (similar to Vox screech), thus why I raised it a bit. I can lower it to 3 or 4 if that's the consensus.

**Comment by Necaladun** [comment]
I'd really prefer we err on the side of screams and other annoying sounds have a longer delay.

**Comment by tigercat2000** [comment]
you already know I am not going to support this because the screams really do hurt my ears

**Comment by Nienhaus** [comment]
:+1:

**Comment by fludd12** [comment]
I've never actually been much of a fan of Goon's screams, I kind of like our willhelm style short screams.

On the other hand, racial and gender based screams.

**Comment by NTSAM** [comment]
Is it possible to have an option to hear "simple screams" (as in the wilhelms we got) opposed to the MYAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAHHHHHH ones?

**Comment by Fox-McCloud** [comment]
Is it possible? Yes. Am I going to spend the time making yet another preference switch just for screams? No.

If someone else wants to, after this, they can do so, but I hate prefs.

**Comment by taukausanake** [comment]
Oh wow. I listened to the screams. I really don't like them. I mean they are good screams but they just don't sound good for SS13 in my opinion. I like the scream that we already have. Maybe that's just the purist in me talking though. That tends to be the case.

@Fox-McCloud I want to say five seconds is good for the 2-3 second long screams. Maybe four but need to hear it in practice first

**Comment by TullyBurnalot** [comment]
:+1: on the monkey and robot screams.

As for the actual male/female screams, I'm afraid I'm gonna have to :-1: it. Maybe I'm just used to our current screaming, but the male/female screams just don't... fit. I cannot honestly say I'd like hearing them in regular gameplay.

**Comment by Spacemanspark** [comment]
I would honestly just add another preference. I like the sounds, but they are rather earshattering.

**Comment by Gummin** [comment]
>:+1: on the monkey and robot screams.

>As for the actual male/female screams, I'm afraid I'm gonna have to :-1: it. Maybe I'm just used to our current screaming, but the male/female screams just don't... fit. I cannot honestly say I'd like hearing them in regular gameplay.

^This.

**Comment by Fox-McCloud** [comment]
You're not going to get a preference for this from me--I don't have SQL set up, and I despise creating preference toggles.

**Comment by Fox-McCloud** [comment]
small updated:

Did a direct, in-game comparison between the goonscream playing at 80 and the "eee" scream sound playing at 100--the volume is roughly the same--if anything the goonscreams are a *hair* bit quieter.

**Comment by KasparoVy** [comment]
Fuck yes, this is going to be so wicked. Really REALLY looking forward to this! SERIOUSLY! :+1: :x: :100:

**Comment by TheDZD** [comment]
I'm honestly kind of on the fence about the male/female screams, kind of prefer the current ones' comic-ness.

**Comment by taukausanake** [comment]
I'm with DZD here. Other screams are fine but I like the current screams for humanoids

**Comment by KasparoVy** [comment]
Lmao these goon screams are pretty damn comical if you ask me, idk what the argument is

The sound quality is so much better than the ones we have
I just don't understand

I guess it's just.. preference? Or something?

**Comment by tigercat2000** [comment]
@KasparoVy Yes, the quality is better- But the female scream is very high-pitched, and loud. If it wasn't so high-pitched, it would be much less annoying, but it hurts my ears as-is.

**Comment by Necaladun** [comment]
No from me, the pitch and just....general sound of it, I find damn annoying.

I don't think there's any way to be objective with something like this, as preference is purely subjective.

File: code/modules/mob/living/carbon/human/species/station.dm [hunk 1]
```diff
@@ -231,6 +231,9 @@
 	flesh_color = "#808D11"
 
 	reagent_tag = PROCESS_ORG
+	scream_verb = "shrieks"
+	male_scream_sound = 'sound/voice/shriek1.ogg'
+	female_scream_sound = 'sound/voice/shriek1.ogg'
 
 	suicide_messages = list(
 		"is attempting to bite their tongue off!",
```
Comment (Crazylemon64) [inline code/modules/mob/living/carbon/human/species/station.dm]: Could IPCs get the robot scream?

Comment (Spacemanspark) [inline code/modules/mob/living/carbon/human/species/station.dm]: YES.

**Comment by Spacemanspark** [comment]
:+1:

**Comment by Fox-McCloud** [comment]
IPC screams are now the same as cyborg

**Comment by LightningLynx** [comment]
:+1: The current ones are dull and hardly sound like screams, being a quiet little "eee." These sound like actual screams (and are way more hilarious)

If anything, maybe the current screams can have their own emote or just be the voluntary scream, and this be involuntary only. But the current ones in comparison just suck. If they're bothersome to people when spammed, well, don't spam them? Vox skree spamming is an issue too but that doesn't mean skree needs to be replaced with a quiet, boring little noise like this one.

**Comment by NTSAM** [comment]
It shouldn't be ear-shatteringly loud, people want to play a videogame and not go deaf.

**Comment by Sky-Ping** [comment]
Monkey Screams

+1+1+1+1+1+1+1+1
+1

**Comment by Fox-McCloud** [comment]
>It shouldn't be ear-shatteringly loud, people want to play a videogame and not go deaf.

The volume for them is lower than the current screams--when you factor in the lower voluem they're played at, the objective decibels/volume of the scream is the same as the current one.

**Comment by Fullofskittles** [comment]
:+1: I like the new screams. I also like the variations in pitch/gender. Is it possible that we might get automatic screaming when players are conscious and in extreme amounts of pain? It just seems like with a longer cooldown you wont be able to scream much at all when you actually need help and cant do anything else. Either way, i like these!

**Comment by Ssirius** [comment]
I love everything about it, Goon has great sound design and i do find Goon screams to be a lot more entertaining.

Borg/IPC screams + Age pitch adjustment + Monkey screams + A longer CD on screams = Whats not to love?

:+1: :100:

**Comment by LightningLynx** [comment]
If they're ear shattering loud, you might want to turn your volume down. These screams have been in Goon for a long time, and they've been doing just fine.

**Comment by TheDZD** [comment]
Seems like a lot of people want this, and it's largely just a subjective thing, so.

**Pull request merged**